docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+Fetched 11.6 kB in 0s (47.3 kB/s)             
+Selecting previously unselected package postgresql-client.
+(Reading database ... 117863 files and directories currently installed.)
+Preparing to unpack .../postgresql-client_16+257build1.1_all.deb ...
+Unpacking postgresql-client (16+257build1.1) ...
+Setting up postgresql-client (16+257build1.1) ...
+Scanning processes...                                                                                                                                                                    
+Scanning linux images...                                                                                                                                                                 
+
+Running kernel seems to be up-to-date.
+
+No services need to be restarted.
+
+No containers need to be restarted.
+
+No user sessions are running outdated binaries.
+
+No VM guests are running outdated hypervisor (qemu) binaries on this host.
+```
+
+We then prepare a script with the information on how to reach and access the database hosted in Pukki.
+On `terminal_C` we type the following:
+```
+$ nano db_cred.sh
+```
+We see a blinking cursor on the top-left corner of the window, indicating that the terminal is ready to receive our input.
+We write in the terminal what follows:
+```
+#!/bin/bash
+export DB_HOST="5.6.7.8"
+export DB_PORT="5432"
+export DB_NAME="pipeline_db"
+export DB_USER="db_admin"
+export PGPASSWORD='db_password'
+```
+When we are done writing, we press the key combination `CTRL + X`.
+From the terminal we are asked if we want to save our changes, to which we reply by pressing the key `y`.
+Finally, it is given to us a chance to modify the name of the file in which our text is saved.
+We are happy with the current name of the file, so we just type `Enter` key, and we are back to our typical terminal view.
+
+Let's now test the access to the database.
+On `terminal_C` we run the following two commands:
+```
+$ source db_cred.sh
+$ psql -h "$DB_HOST" -p "$DB_PORT" -U "$DB_USER" -d "$DB_NAME"
+```
+After entering the second command we get the following prompt, which indicates successful connection to the database:
+```
+psql (16.6 (Ubuntu 16.6-0ubuntu0.24.04.1), server 17.2 (Debian 17.2-1.pgdg110+1))
+WARNING: psql major version 16, server major version 17.
+         Some psql features might not work.
+SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
+Type "help" for help.
+
+pipeline_db=>
+```
+To go back to our virtual machine we just type `exit` and press `Enter` key.
+
+Now that we can communicate with the database, we prepare it to host the data that we will send to it when processing our images.
+On `terminal_C` we run the following command:
+```
+$ psql -h "$DB_HOST" -p "$DB_PORT" -U "$DB_USER" -d "$DB_NAME" -c "CREATE TABLE IF NOT EXISTS log_records (timestamp varchar(25) primary key, negated_image_name text, negated_image_hash text)"
```

Comment:
Please cut this complex command is several lines with `\` to show better without horizontal scrolling.

Reply:
Fixed in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/bc111024bc06c203eea12d7e3af4fe02c8fe2da3